tests/test_stubs.py

```diff
@@ -56,7 +55,26 @@ def test_already_converted(base_project, tmp_path):
     mzml = mzml_dir / "a.mzML.gz"
     mzml.touch()
     old = mzml.stat()
-    test_no_aggregate(base_project=base_project, tmp_path=tmp_path)
+
+    config, *_ = base_project
+    cmd = ["nextflow", "run", "main.nf"] + config
+    subprocess.run(cmd, check=True)
 
     assert old == mzml.stat()
     assert (mzml_dir / "b.mzML.gz").exists()
+
+
+def test_force_convert(base_project, tmp_path):
+    """Test that we can force files to be converted again."""
+    mzml_dir = (tmp_path / "mzml" / "subdir")
+    mzml_dir.mkdir(parents=True)
+    mzml = mzml_dir / "a.mzML.gz"
+    mzml.touch()
+    old = mzml.stat()
+
+    config, *_ = base_project
+    cmd = ["nextflow", "run", "main.nf", "--msconvert.force", "true"] + config
+    subprocess.run(cmd, check=True)
+
+    assert old != mzml.stat()
```

Comment:
Non-blocking question: I didn't run this, but shouldn't old == mzml.stat() since that is what you set it to?

Reply:
So in the above test (`test_already_converted`), this is correct. This test is actually for the opposite---in the case that use the `msconvert.force` param, I want to make sure that the existing file is overwritten.